crates/pop-contracts/src/lib.rs

```diff
@@ -30,7 +30,7 @@ pub use up::{
 };
```

Comment:
The changes in the `pop-contracts` files are only for the moving of `parse_account` into the common crate.